### PR TITLE
PM-1064 Deploy MR1 features in the SUD app

### PR DESCRIPTION
**Pull request by andriy-morgunov-epam**
https://1mojio.atlassian.net/browse/PM-1064

⁉️ What does this PR do?

- Appended new properties that Platform supports for Vehicle

Seatbelt
Seats
BrakeFluid
NextService
HybridBattery
HybridEngine

📖 Any background context you want to provide?

- None

### CODE REVIEW

**Review by stepanrazin — APPROVED** [review]

File: MojioSDK.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -603,7 +621,7 @@
 				BC879B581CBEFFDD00F9177B /* RPM.swift */,
 				BC879B5A1CBEFFDD00F9177B /* Speed.swift */,
 				BC879B4A1CBEFFDD00F9177B /* TimePeriod.swift */,
-				37BC6C8223129E4B0054B420 /* EngineOilTemperature.swift */,
```
Comment (pauluhn) [inline MojioSDK.xcodeproj/project.pbxproj:606]: #? intended?

File: MojioSDK/Models/Vehicles/BrakeFluid.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/******************************************************************************
+ * Moj.io Inc. CONFIDENTIAL
+ * 2017 Copyright Moj.io Inc.
```
Comment (pauluhn) [inline MojioSDK/Models/Vehicles/BrakeFluid.swift]: #req update copyright

File: MojioSDK.xcodeproj/project.pbxproj (hunk 1 above)
Comment (andriy-morgunov-epam) [reply MojioSDK.xcodeproj/project.pbxproj:606]: @pauluhn This is Xcode project file editing side effect. In line 614 you can see the new line for "EngineOilTemperature". So this line changed the position, it wasn't deleted

**Comment by andriimakukha** [comment]
@andriy-morgunov-epam Copyrights were updated in scope https://github.com/mojio/mojio-ios-sdk/pull/342.

**Review by andriimakukha — APPROVED** [review]

File: MojioSDK/Models/Vehicles/BrakeFluid.swift (hunk 2 above)
Comment (andriy-morgunov-epam) [reply MojioSDK/Models/Vehicles/BrakeFluid.swift]: @pauluhn Copyrights were updated

**Review by pauluhn — APPROVED** [review]

**Pull request merged**